main.css

```diff
@@ -0,0 +1,133 @@
+body {
+  background-color: hsl(204, 43%, 93%);
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  font-family: Karla;
+  margin: 0;
+}
+
+main {
+  max-width: 636px;
+  border-radius: 16px;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: auto; /* why we need to use margin here, if i do not use margin then the border radius does not apply*/
+  box-shadow: 0px 0px 20px rgb(0 0 0 / 20%);
+}
+
+.intro {
+  justify-content: center;
+  background-color: white;
+  padding: 0 40px;
+  height: 216px;
+}
+
+.main-text {
+  margin-top: 38px;
+  font-size: 24px;
+  color: #2ab2af;
+}
+
+.second-heading {
+  color: #c0df34;
+  font-size: 18px;
+  margin: 24px 0px 0px;
+}
+
+.third-heading {
+  color: #96a6bd;
+  font-size: 15px;
+  margin: 10px 0px 0px;
+  line-height: 26px;
+}
+
+.sub-part {
+  display: flex;
+  flex-direction: row;
+  height: 258px;
```

Comment:
By adding height container will no longer be flexible if we add a thing later in it.

Reply:
removed.